mirrord/layer/src/tcp_steal.rs

```diff
@@ -111,6 +111,8 @@ pub struct TcpStealHandler {
 }
 
 impl TcpHandler for TcpStealHandler {
+    const IS_STEAL: bool = true;
```

Comment:
Mom always told me that stealing is wrong!

Reply:
What did she say about mirroring?